### ISSUE TITLE
pre-built mutation annotated tree README doesn't link to correct references

### ISSUE THREAD

**Issue by jbloom** [yatisht/usher#322]
The pre-built mutation annotated trees at http://hgdownload.soe.ucsc.edu/goldenPath/wuhCor1/UShER_SARS-CoV-2/ has a README that says to cite Lanfear (2020) and Turakhia et al (2021).

But it seems like [McBroome et al (2021)](https://academic.oup.com/mbe/article/38/12/5819/6361626) is also a relevant reference for these pre-built trees (maybe the most relevant one?) and should be listed in README too?

**Comment by AngieHinrichs** [comment]
Yes, thank you for pointing that out @jbloom!  I will update it.
